style.css

```diff
@@ -0,0 +1,719 @@
+body {
+	margin: 0;
+	font-family: Montserrat;
+}
+.collection__img-mob {
+	display: none;
+}
+.header {
+	position: absolute;
+	z-index: 3;
+	width: 100%;
+}
+.header__inner {
+	display: flex;
+	justify-content: space-between;
+}
+.header__logo {
+	font-size: 30px;
+	letter-spacing: 3px;
+	margin: 6px 52px;
+	z-index: 2;
+	color: #FFFFFF;
+}
+.header__menu {
+	display: flex;
+	flex-direction: column;
+	height: 92px;
+	width: 92px;
+	justify-content: center;
+	align-items: center;
+	background: rgb(0, 0, 0, 0.05);
+}
+.header__menu-item {
+	height: 3.48px;
+	width: 34.7px;
+	background: black;
+	margin-bottom: 5.96px;
+}
+.header__language {
+	position: absolute;
+	left: 27%;
+	top: 42px;
+	font-style: normal;
+	font-weight: 500;
+	font-size: 20px;
+	color: #FFFFFF;
+	z-index: 2;
+}
+.header__vector {
+	position: absolute;
+	left: 29.5%;
+	top: 54px;
+	z-index: 2;
+}
+.first {
+	width: 100%;
+	display: grid;
+	grid-template-columns: 32% 68%;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5598887)